src/components/forms/DevconGrantsForm.tsx

```diff
@@ -0,0 +1,1015 @@
+            color='brand.paragraph'
+            fontSize='input'
+            h='150px'
+            mt={3}
+            {...register('sponsorshipDetails', {
+              required: true,
+              maxLength: 32768
+            })}
+          />
+
+          {errors?.sponsorshipDetails?.type === 'required' && (
+            <Box mt={1}>
+              <PageText as='small' fontSize='helpText' color='red.500'>
+                Event details are required.
+              </PageText>
+            </Box>
+          )}
+          {errors?.sponsorshipDetails?.type === 'maxLength' && (
+            <Box mt={1}>
+              <PageText as='small' fontSize='helpText' color='red.500'>
+                Event details cannot exceed 32768 characters.
+              </PageText>
+            </Box>
+          )}
+        </FormControl>
+
+        <FormControl id='project-description-control' mb={8}>
+          <FormLabel htmlFor='projectDescription' mb={1}>
+            <PageText fontSize='input'>What is your project about?</PageText>
+          </FormLabel>
+
+          <PageText as='small' fontSize='helpText' color='brand.helpText'>
+            If you have a project you&apos;d like to discuss, give us a short summary of what you
+            are hoping to accomplish. Just a paragraph will do.
+          </PageText>
+
+          <Textarea
+            id='projectDescription'
+            bg='white'
+            borderRadius={0}
+            borderColor='brand.border'
+            color='brand.paragraph'
+            fontSize='input'
+            h='150px'
+            mt={3}
+            {...register('projectDescription', {
+              maxLength: 32768
+            })}
+          />
+
+          {errors?.projectDescription?.type === 'maxLength' && (
+            <Box mt={1}>
+              <PageText as='small' fontSize='helpText' color='red.500'>
+                Project description cannot exceed 32768 characters.
+              </PageText>
+            </Box>
+          )}
+        </FormControl>
+
+        <Flex direction={{ base: 'column', lg: 'row' }}>
+          <Controller
+            name='eventType'
+            control={control}
+            defaultValue={{ value: '', label: '' }}
+            render={({ field: { onChange } }) => (
+              <FormControl id='event-type-control' isRequired mb={8} mr={{ md: 12 }}>
+                <FormLabel htmlFor='eventType'>
+                  <PageText display='inline' fontSize='input'>
+                    What type of event is this?
+                  </PageText>
+                </FormLabel>
+
+                <Select
+                  id='eventType'
+                  options={EVENT_TYPE_OPTIONS}
+                  onChange={onChange}
+                  components={{ DropdownIndicator }}
+                  placeholder='Select'
+                  closeMenuOnSelect={true}
+                  selectedOptionColor='brand.option'
+                  chakraStyles={chakraStyles}
```

Comment:
there's a tiny UI bug with the `@` char of the twitter field below because that car has a `zIndex` value of 9, higher than the dropdown container (you can see the @ below the 'Hackaton' option)

<img width="417" alt="Screen Shot 2022-03-04 at 13 39 55" src="https://user-images.githubusercontent.com/948922/156803487-b3711818-020d-4afc-b1e0-faba0caddfa7.png">

this can be fixed by editing `chakraStyles` and adding a higher zIndex value for `container`, like 99

```
container: provided => ({
  ...provided,
  background: 'white',
  borderColor: 'brand.border',
  color: 'brand.paragraph',
  zIndex: 99
}),
...
```

Reply:
my bad, zIndex should be added to `menu` instead of container